### ISSUE TITLE
semi rule doesn’t warn for missing semicolon in throw statements

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#1873]
The following gives no warning when the `semi` rule is configured with `"semi": [2, "always"]`

```js
throw new Error('foo')
```

**Comment by lo1tuma** [comment]
Working on it.
